test/fixtures/linter/autofix/snowflake/002_previously_parse_tree_damaging/before.sql

```diff
@@ -5,8 +5,7 @@ select
 from foo.bar
 where
 split(foo, '|')[2] REGEXP '^\\d+\\-\\d+\\-\\d+ \\d+\\:\\d+$'
-OR
-foo IN ('BAR','FOO')
+OR foo IN ('BAR','FOO')
```

Comment:
Why'd we change the _input_ file? (Probably not important, just curious.)

Reply:
I think the current way of indenting this I would consider wrong, and so didn't want to codify it